app/src/main/res/values/styles.xml

```diff
@@ -572,4 +572,12 @@
         <item name="android:backgroundTint">?attr/savedSiteEditTextUnderlineColor</item>
     </style>
 
+    <style name="Theme.AppCompat.Transparent.NoActionBar" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
We already have an style in `themes.xml` for transparent activities `<style name="AppTheme.Transparent">`. Any reason to have it duplicated here?

Reply:
I tried that style initially but it has a transition when you start the activity that doesn't look that great:

https://user-images.githubusercontent.com/3471025/133599644-bb05d90e-90f1-42fb-b67c-0b89b0328c80.mp4